testsuite/integration/basic/src/test/java/org/wildfly/test/integration/jmx/rbac/AbstractJmxAccessFromDeploymentWithRbacTest.java

```diff
@@ -226,8 +221,8 @@ private ModelNode createSuperUserRoleMapping(String userName) {
                     .append(ROLE_MAPPING, "SuperUser")
                     .append(INCLUDE, "user-" + userName);
             ModelNode addSuperUserInclude = Util.createAddOperation(addr);
-            addSuperUserInclude.get(NAME, userName);
-            addSuperUserInclude.get(TYPE, "USER");
+            addSuperUserInclude.get(NAME).set(userName);
+            addSuperUserInclude.get(TYPE).set("USER");
```

Comment:
I just realized I forgot to look at why these tests were successful even though the setup task failed.